monitoring/zabbix_host.py

```diff
@@ -415,7 +426,7 @@ def main():
     module = AnsibleModule(
         argument_spec=dict(
             server_url=dict(type='str', required=True, aliases=['url']),
-            login_user=dict(rtype='str', equired=True),
+            login_user=dict(rtype='str', required=True),
```

Comment:
I pushed this fix immediately (also see rtype instead of type) as we are about to freeze, please rebase.